pkgs/development/python-modules/qiskit-aqua/default.nix

```diff
@@ -0,0 +1,79 @@
+{ lib
+, pythonOlder
+, buildPythonPackage
+, fetchFromGitHub
+, cvxopt
+, dlx
+, docplex
+, fastdtw
+, h5py
+, networkx
+, numpy
+, psutil
+, pyscf
+, qiskit-terra
+, qiskit-ignis
+, quandl
+, scikitlearn
+  # Optional extras
+, pytorch
+  # Check Inputs
+, qiskit-aer
+, stestr
+, parameterized
+}:
+
+buildPythonPackage rec {
+  pname = "qiskit-aqua";
+  version = "0.6.2";
+
+  disabled = pythonOlder "3.5";
+
+  # Pypi's tarball doesn't contain tests
+  src = fetchFromGitHub {
+    owner = "Qiskit";
+    repo = pname;
+    rev = version;
+    sha256 = "1jnnk5fsaxc5m7gsdjny6hwmc1vjcdw3qyvbq714yhf8mccb96va";
+  };
+
+  propagatedBuildInputs = [
+    cvxopt
+    docplex
+    dlx # Python Dancing Links package
+    fastdtw
+    h5py
+    networkx
+    numpy
+    psutil
+    pyscf
+    pytorch # Optional extra
+    qiskit-terra
+    qiskit-ignis
+    quandl
+    scikitlearn
+  ];
+
+  # Tests take about 12 mins to run on 8-core, parallelized. Not sure if worth running or not.
```

Comment:
unless using something like `pytest-xdist`, these tests will be single threaded anyway, so yes, they are worth. However, if you can conditionally disable the slow ones, that is also acceptable

Reply:
stestr theoretically runs them multi-threaded, unless nix does some behind-the-scenes magic not to make that happen.